kinder/pkg/constants/constants.go

```diff
@@ -126,4 +126,7 @@ var (
 const (
 	// KinderVersion is the kinder CLI version
 	KinderVersion = "0.1.0-alpha.3"
+
+	// DefaultKubeadmConfigVersion indicates the default config API version that kubeadm uses
+	DefaultKubeadmConfigVersion = "v1beta2"
```

Comment:
Uhmm, we can't have a DefaultKubeadmConfigVersion, because this depends by the kubeadm version we are using.
Eg. for kubeadm v1.22 it is v1beta3, for older version v1beta2 (and for older older versions v1beta1, but I'm not sure they are still in the support skew)